src/SpeedyWeather.jl

```diff
@@ -17,7 +17,7 @@ import Adapt: Adapt, adapt, adapt_structure
 
 # INPUT OUTPUT
 import TOML
-import Dates: Dates, DateTime
+import Dates: Dates, DateTime, Second, Minute, Hour, Day, value
```

Comment:
I would not import `value` into scope simply because it's too common a term and therefore to easy to overwrite and cause unexpected errors, one can do `time_scale.value` to obtain the `Int64` behind a `time_scale::Second` (or any other time period).

Reply:
addressed with [fd2b5e2](https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/418/commits/fd2b5e24547033dd18cffaca922ab7b9f559ad6f)